crates/uv/src/commands/project/remove.rs

```diff
@@ -143,3 +136,22 @@ pub(crate) async fn remove(
 
     Ok(ExitStatus::Success)
 }
+
+/// Emit a warning if a dependency with the given name is present as any dependency type.
+fn warn_dependency_types(name: &PackageName, pyproject: &PyProjectTomlMut) {
```

Comment:
Should this just be part of the `anyhow::bail!`? It took me a sec to realize that this running after we've been unable to find the dependency in some _other_ group. I wonder if there's a way to structure the code such that the data flow is clearer. Maybe just a matter of expanding the documentation.

Reply:
Renamed to `warn_if_present` and updated the docs.